### DISCUSSION TITLE
vhd: Waiting for WinRM to become available

### DISCUSSION THREAD

**Discussion by celenn** [actions/virtual-environments#5082] [General questions]
Hi 
I have been using virtual environment yaml pipeline to create image on the Azure. However the pipeline exits with the error "==> vhd: Timeout waiting for WinRM. after 34 minutes. 
I would be more than happy if I get any suggestions on the issue.

Thanks

**Comment by al-cheb** [comment]
https://github.com/actions/virtual-environments/issues/1119#issuecomment-649196766
